run/john.conf

```diff
@@ -42,9 +42,10 @@ SingleRules = Single
 # Default batch mode Wordlist rules
 BatchModeWordlistRules = Wordlist
 
-# Default wordlist mode rules when not in batch mode (if any)
-# If this is set and you want to run once without rules, use --rules:none
-#WordlistRules = Wordlist
+# Default wordlist mode rules when not in batch mode (if any).  If this is
+# changed from 'none' to have default rules applied, and you DO want to run
+# once without rules, use --rules:none on the command line.
+WordlistRules = none
```

Comment:
I think this might be a regression. We should compare using `-w` vs. `-w -ru:none` for some of the fastest formats.

Reply:
Agreed.   I know the 'results' should be the same.  BUT there could be certain additional issues, going from 'no' rules, to a 'dummy' rule. 

I am glad question this showed up in the PR.  This change (and  the change to ``` DefaultIncrementalUTF8 = ASCII ``` ) are the 2 which troubled me the most.  Keeping the logic 'dictates' them, BUT they certainly give me the chills, thinking there is something not quite right.  The date formats, setting same logic for NULL as for "" is no big issue.  It is only at the 'exceptional event' of a log, and an extra if is no big deal.

Reply:
It should be easy to change the code to not enable rules at all for a null string.

Reply:
That is probably a better solution, than using rules:none (much like the handling of date logic).  I will work on making it so, then it should be 'identical' both in logic and implementation, where with the patch as it stands it is only equal in logic.

Is the change to ``` DefaultIncrementalUTF8 = ASCII ``` OK?  That was about the only other thing that made the hair on the back of my neck stand up a bit.

Reply:
> Is the change to `DefaultIncrementalUTF8 = ASCII` OK? That was about the only other thing that made the hair on the back of my neck stand up a bit.

Yes, given that's the default it's actually better - because it's harder to miss!